tests/test.spec.ts

```diff
@@ -47,4 +47,17 @@ describe('templates can be instantiated', () => {
       })
     })
   })
+  describe('with custom engineType', () => {
+    Object.values(PrismaTemplates.Templates).forEach((Template) => {
+      it(Template.metadata.name, () => {
+        const template = new Template({
+          datasourceProvider: 'mysql',
+          repositoryOwner: 'prisma',
+          repositoryHandle: 'templates-node',
+          engineType: 'library',
+        })
+        expect(template).toMatchSnapshot()
```

Comment:
This is a 14k snapshot, its massive, can we do a non-snapshot based assertion here?

Reply:
Yeah we could, but the formatting of the resulting file is pretty helpful. I changed the repository owner/handle and engineType tests to check against the output of a single file for every template instead of every file, metadata, and artifact for every template. This reduces the snapshot by ~5k lines which takes it to about the same size it was before the new features introduced this week.